site/content/docs/5.2/examples/form/index.html

```diff
@@ -233,7 +231,7 @@ <h2 class="visually-hidden">Sitemap &amp; information</h2>
       <li><a class="nav-link" href="#">Privacy</a></li>
       <li><a class="nav-link" href="#">Cookie policy</a></li>
       <li><a class="nav-link" href="#">Accessibility statement</a></li>
-      <li><a class="nav-link active" id="test" href="#" aria-current="page">Become a reseller</a></li>
```

Comment:
Active should stay even if it doesn't have any real rendering

Reply:
But if it stays it's not valid from an accessibility point of view since it's convey informations only by color.